src/hooks/cartReducer.js

```diff
@@ -0,0 +1,39 @@
+export const cartReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_TO_CART":
```

Comment:
- Same here, keep all the action types you are using for switch cases in a separate file and use them here.